lib/io/fileutils/fileutils.go

```diff
@@ -248,7 +248,7 @@ func RemoveFirstLine(filePath string) error {
 	defer file.Close()
 
 	// Create a temporary file to write the updated content
-	tempFile, err := os.CreateTemp("", "temp")
+	tempFile, err := os.CreateTemp("/hab/tmp", "temp")
```

Comment:
pass `/hab/tmp` as an argument 
Eg:
```
func RemoveFirstLine(dir string, filePath string) error {}
```

Reply:
I tried that first (just as we spoke in Teams), but a lot more changes are required because of the FileUtils interface.